### PR TITLE
WS-012 Style API

### PR DESCRIPTION
**Pull request by Eyrim**

### CODE REVIEW

**Comment by Eyrim** [comment]
image

**Comment by Eyrim** [comment]
image

**Comment by Eyrim** [comment]
image

File: src/main/resources/static/css/api.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.page-container {
+    background-color: #42BFDD;
```
Comment (AndrewYHuang) [inline src/main/resources/static/css/api.css:2]: Just a thought here about using colour pallettes

https://github.com/charlottemothersole/WhaleSpotting-Java/pull/10#discussion_r1351944697

**Review by AndrewYHuang — APPROVED** [review]
Looks good!

File: src/main/resources/static/css/api.css (hunk 1 above)
Comment (Eyrim) [reply src/main/resources/static/css/api.css:2]: Will update in a future ticket after this merge

**Pull request merged**